src/Tizen.Multimedia.Remoting/MediaController/MediaControlMetadata.cs

```diff
@@ -116,5 +125,150 @@ internal MediaControlMetadata(IntPtr handle)
+        public Size Resolution
+        {
+            get => DecodeResolution(EncodedResolution);
+            set => EncodedResolution = EncodeResolution(value.Width, value.Height);
+        }
+
+        // Native CAPI used only encoded string to gets or sets Season, Episode, Resolution.
+        // But encoded string is not useful for user, so we don't offer as it is.
+        // It'll be used internally.
+        internal string EncodedSeason { get; private set; }
+
+        internal string EncodedEpisode { get; private set; }
+
+        internal string EncodedResolution { get; private set; }
+
+        private static string EncodeSeason(SeriesInformation information)
+        {
+            Native.EncodeSeason(information.Number, information.Title, out string encodedSeason).
+                ThrowIfError("Failed to encode season");
+
+            return encodedSeason;
+        }
+
+        private static string EncodeEpisode(SeriesInformation information)
+        {
+            Native.EncodeEpisode(information.Number, information.Title, out string encodedEpisode).
+                ThrowIfError("Failed to encode episode");
+
+            return encodedEpisode;
+        }
+
+        private static string EncodeResolution(int width, int height)
+        {
+            Native.EncodeResolution((uint)width, (uint)height, out string encodedResolution).
+                ThrowIfError("Failed to encode resolution");
+
+            return encodedResolution;
+        }
+
+        private static SeriesInformation DecodeSeason(string encodedSeason)
+        {
+            int number = 0;
+            string title = null;
+
+            if (encodedSeason != null)
+            {
+                Native.DecodeSeason(encodedSeason, out number, out title).
+                    ThrowIfError("Failed to decode season");
+            }
+
+            return new SeriesInformation(number, title);
+        }
+
+        private static SeriesInformation DecodeEpisode(string encodedEpisode)
+        {
+            int number = 0;
+            string title = null;
+
+            if (encodedEpisode != null)
+            {
+                Native.DecodeEpisode(encodedEpisode, out number, out title).
+                    ThrowIfError("Failed to decode episode");
+            }
+
+            return new SeriesInformation(number, title);
+        }
+
+        private static Size DecodeResolution(string encodedResolution)
+        {
+            uint width = 0, height = 0;
+
+            if (encodedResolution != null)
+            {
+                Native.DecodeResolution(encodedResolution, out width, out height).
+                    ThrowIfError("Failed to decode resolution");
+            }
+
+            return new Size((int)width, (int)height);
+        }
+    }
+
+    /// <summary>
+    /// Represents properties for the video series information.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class SeriesInformation
```

Comment:
Sorry, in this case your decision more good for usability, becuase it provided as property in MediaControlMetadata class please use `struct`

Reply:
OK I'll change it.

Reply:
``` c#
 MediaControlMetadata metadata = .... 
 metadata.Episode.Title = "a new title"; // it should not allowed,  struct is not allowed, class is allow
```

Reply:
`Title` and `Number` property of `SeriesInformation` are read only. So, even if it's class, it does not allowed. Of course `struct` cannot try it at all.

Reply:
Oh, you are right. class also good for usability